stdlib/ssl.pyi

```diff
@@ -369,6 +369,7 @@ class SSLSocket(socket.socket):
     def compression(self) -> str | None: ...
     def get_channel_binding(self, cb_type: str = "tls-unique") -> bytes | None: ...
     def selected_alpn_protocol(self) -> str | None: ...
+    @deprecated("Deprecated in 3.10. Use ALPN instead.")
     def selected_npn_protocol(self) -> str | None: ...
```

Comment:
In cases like this, we actually make the deprecation Python-version dependent. I.e.:

```suggestion
    if sys.version >= (3, 10):
        @deprecated("Deprecated in 3.10. Use ALPN instead.")
        def selected_npn_protocol(self) -> str | None: ...
    else:
        def selected_npn_protocol(self) -> str | None: ...
```

Same for the other methods below.